packagingtest/centos7/systemd/Dockerfile

```
@@ -72,19 +67,14 @@ RUN sed -ri 's/^session\s+required\s+pam_loginuid.so$/session optional pam_login
         sed -ri 's/requiretty/!requiretty/' /etc/sudoers && \
         echo 'root:docker.io' | chpasswd
 
-# Set default locale to 'UTF-8' for the test execution environment
-# Per https://github.com/CentOS/sig-cloud-instance-images/issues/71
-RUN sed -ri 's/langs=en_US.UTF-8/langs=en_US.utf8/' /etc/yum.conf && \
-    yum reinstall -y glibc-common
-
 RUN yum -y install nc net-tools
 
 ENV container docker
 
 RUN yum -y update; \
     yum -y install systemd; yum clean all;
 
-RUN cd /lib/systemd/system/sysinit.target.wants/; ls -1 | grep -v systemd-tmpfiles-setup.service | xargs rm; \
+RUN cd /lib/systemd/system/sysinit.target.wants/; ls -1 | grep -v systemd-tmpfiles-setup.service | xargs rm; \rm -f /lib/systemd/system/multi-user.target.wants/*; \
```

Per previous comment (https://github.com/StackStorm/st2packaging-dockerfiles/pull/74#discussion_r359076082), please don't change this line.

```suggestion
RUN cd /lib/systemd/system/sysinit.target.wants/; ls -1 | grep -v systemd-tmpfiles-setup.service | xargs rm; \
```

---

done

---

This still needs to be addressed.
We don't need `rm -f /lib/systemd/system/multi-user.target.wants/*;` line in CentOS Dockerfiles